README.md

```diff
@@ -506,6 +507,19 @@ const result = words(str)
 //    ^ ['20', 'some', 'Very', 'weird', 'String']
 ```
 
+### truncate
+
+This function truncates string if it's longer than the given maximum string length. The last characters of the truncated string are replaced with the omission string which defaults to "...".
+
+
+```ts
+import { truncate } from 'string-ts'
```

Comment:
truncate should come before words in the whole README as every method is sorted alphabetically

Reply:
fixed in [163db6d](https://github.com/gustavoguichard/string-ts/pull/41/commits/163db6ddda19b77135070a2bed8b2b5355752861)